bin/purge_expired_tokens.js

```diff
@@ -25,7 +25,7 @@ config.set('db.autoUpdateClients', false);
 program
   .version(package.version)
   .option('-c, --config [config]', 'Configuration to use. Ex. dev')
-  .option('-p, --pocket-id <pocketId>', 'Pocket Client Id. These tokens will not be purged.')
+  .option('-p, --pocket-id <pocketId>', 'Pocket Client Ids. These tokens will not be purged. (CSV)')
```

Comment:
nit: Could update example command line in https://github.com/mozilla/fxa-oauth-server/pull/419/files#diff-ad4b135c187bb7e96c8db4aa28dfa456L14